lib/everypolitician/popolo.rb

```diff
@@ -30,18 +30,27 @@ def organizations
       end
     end
 
-    class People
+    class Collection
       include Enumerable
 
       attr_reader :documents
 
-      def initialize(documents)
-        @documents = documents.map { |p| Person.new(p) }
-      end
-
       def each(&block)
         documents.each(&block)
       end
+
+      def -(other)
+        ids = documents.map { |d| d.id }
+        other_ids = other.documents.map { |d| d.id }
+        different_ids = ids - other_ids
+        different_ids.map { |id| (documents + other.documents).find { |d| d.id == id } }
```

Comment:
Why do we need to `find` against `other.documents` here? Surely in `documents - other` nothing can end up in the result that was only in `other`? 

Also, on the larger legislatures (e.g Turkey), anything that involves `find` in a loop tends to be very slow (we effectively have to do 6000 traversals of (on average) half of a 6000 element list). This whole thing seems like it might be better as something along the lines of:

```ruby
  other_ids = Set.new other.documents.map(&:id)
  documents.reject { |d| other_ids.include? d.id }
```